README.md

```diff
@@ -34,3 +55,95 @@ already posses acces token to go through kong. The actual token verification is
 If you're using `luarocks` execute the following:
 
      luarocks install kong-oidc
+
+You also need to set the `KONG_CUSTOM_PLUGINS` environment variable
+
+     export KONG_CUSTOM_PLUGINS=oidc
+     
+## Usage
+
+### Parameters
+
+| Parameter | Default  | Required | description |
+| --- | --- | --- | --- |
+| `name` || true | plugin name, has to be `oidc` |
+| `config.client_id` || true | OIDC Client ID |
+| `config.client_secret` || true | OIDC Client secret |
+| `config.discovery` | https://.well-known/openid-configuration | false | OIDC Discovery Endpoint (`/.well-known/openid-configuration`) |
+| `config.scope` | oidc | false| OAuth2 Token scope. To use OIDC it has to contains the `oidc` scope |
+| `config.ssl_verify` | false | false | Enable SSL verification to OIDC Provider |
+| `config.session_secret` | | false | Additional parameter, which is used to encrypt the session cookie. Needs to be random |
+
+### Enabling
+
+To enable the plugin only for one API:
+
+```
+POST /apis/<api_id>/plugins/ HTTP/1.1
+Host: localhost:8001
+Content-Type: application/x-www-form-urlencoded
+Cache-Control: no-cache
+
+name=oidc&config.client_id=kong-oidc&config.client_secret=29d98bf7-168c-4874-b8e9-9ba5e7382fa0&config.discovery=http%3A%2F%2Fkeycloak%3A8080%2Fauth%2Frealms%2Fmaster%2F.well-known%2Fopenid-configuration
```

Comment:
Better to say oidc_provider and not keycloak (this is in 3 places).

Reply:
In fact this is an example query, so it don't thing it's required to generalize it.

Does `https%3A%2F%2F<oidc_provider>%2F.well-known%2Fopenid-configuration` look better?

Reply:
Yes. Point is that the oidc provider could be something else than keycloak.

Reply:
OK, I changed it. Can you later squash the commits when merging the PR?